needs-redeploy.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import subprocess
+
+if len(sys.argv) != 3:
+    sys.stderr.write(f'''usage: {sys.argv[0]} <deployed-hash> <project>
+
```

Comment:
That's a multi-line comment.  I want a line between the usage and the explanation, that is common (e.g. see `git --help`) and the indentation is because we're in a string literal and indentation would show up in the usage message.

Reply:
The output currently looks like:

```
$ python needs-redeploy.py foo bar baz
usage: needs-redeploy.py <deployed-hash> <project>

outputs 'yes' if <project> deployed as <deployed-hash> needs be
redeployed, else 'no'.
```

Reply:
makes sense